templates/swagger/v1_json.tmpl

```diff
@@ -13959,7 +14106,7 @@
     "parameterBodies": {
       "description": "parameterBodies",
       "schema": {
-        "$ref": "#/definitions/EditBranchProtectionOption"
+        "$ref": "#/definitions/CreateOAuth2ApplicationOptions"
```

Comment:
ok this looks weared!

Reply:
strange indeed, this files generates automatically. No idea why its changing here.

Reply:
It's ok look at: https://github.com/go-gitea/gitea/pull/10437/files#diff-8798bbdc42c2deaa2de7c724ab417e39L12-L13 .. it's because of a "hack" ;)

so you can resolve this :)

Reply:
(didn't noticed until now :))